Modules/_io/_iomodule.c

```diff
@@ -368,6 +367,14 @@ _io_open_impl(PyObject *module, PyObject *file, const char *mode,
         if (blksize_obj == NULL)
             goto error;
         buffering = PyLong_AsLong(blksize_obj);
+        if (buffering > 8192 * 1024)
```

Comment:
The curly brace style here isn't python standard

Moving to standard would be:
```C
if (buffering > 8192 * 1024) {
    buffering = 8192 * 1024
}
if (buffering < DEFAULT_BUFFER_SIZE) {
    buffering =
}
```

although practically, i think Py_MIN and Py_MAX are going to be cleaner here:
buffering = Py_MAX(Py_MIN(buffering, 8192 * 1024), DEFAULT_BUFFER_SIZE);